lib/components/desktop/wallpaper_picker.dart

```diff
@@ -38,20 +39,22 @@ class _WallpaperPickerState extends State<WallpaperPicker>
         StateServiceListener<CustomizationService, WallpaperPicker> {
   final TextEditingController _controller = TextEditingController();
   late TabController tabController;
+  late Future<List<Wallpaper?>?> wallpapers;
 
   @override
   void initState() {
     super.initState();
-    tabController = TabController(length: 3, vsync: this);
+    tabController = TabController(length: 2, vsync: this);
```

Comment:
tabs should be "dynamic" in the sense that they should live in a variable so you can change the items in there and have everything else be properly updated